src/coverlet.console/Program.cs

```diff
@@ -23,9 +22,17 @@ class Program
     {
         static int Main(string[] args)
         {
-            var serviceProvider = GetServiceProvider();
-            var logger = DependencyInjectionExtensions.GetService<ILogger>(serviceProvider);
-            var fileSystem = DependencyInjectionExtensions.GetService<IFileSystem>(serviceProvider);
+            IServiceCollection serviceCollection = new ServiceCollection();
+            serviceCollection.AddTransient<IRetryHelper, RetryHelper>();
+            serviceCollection.AddTransient<IProcessExitHandler, ProcessExitHandler>();
+            serviceCollection.AddTransient<IFileSystem, FileSystem>();
+            serviceCollection.AddTransient<ILogger, ConsoleLogger>();
+            serviceCollection.AddSingleton<IInstrumentationHelper, InstrumentationHelper>();
+            var serviceProvider = serviceCollection.BuildServiceProvider();
```

Comment:
Please avoid `var` if right expression type is not "clear"

Reply:
Done